Examples/DataRepresentation/Containers/TreeContainer.cxx

```diff
@@ -64,7 +65,7 @@ int main(int, char* [])
 
   // Software Guide : BeginCodeSnippet
   using NodeType = int;
-  using TreeType = itk::TreeContainer<NodeType>;
+  using TreeType = itk::TreeContainer< NodeType >;
```

Comment:
For me personally these spaces around an argument are not really necessary. (But I can get used to it.)

Reply:
I have mixed feelings on this.  1. condensed is nice for more content on one line, but 2. My eyesight is really bad, and the spaces do facilitate the mechanics of reading code with spaces.

In reviewing the historical code, the use/disuse of spaces in cases like this was mixed without a definitive common practice.  Either choice was going to make many code changes.

Again the goal is to provide a more easily enforced common standard.

Reply:
Space is _required_ by some compilers when there are multiple embedded templates. For example: ` itk::TreeContainer<std::vector<NodeType>>` throws error on some compilers. Always adding the space (even if it's not required by the compiler) makes things a bit more consistent.

Reply:
Same, I got used to it after a while.

Reply:
@lassoan those spaces were only required pre-C++11, no?

Reply:
I just remember that I had to add those spaces for some compilers. Maybe they are no longer needed.

Reply:
With C++11 (which we require since 5.0) space between closing template angle brackets is no longer required, so `itk::TreeContainer<std::vector<NodeType>>` is fine.

I prefer no spaces, but I have gotten used to spaces since using ITK extensively.